app/controllers/distributions_controller.rb

```diff
@@ -6,18 +6,25 @@ def index
     render layout: 'download'
   end
 
-  # GET /distributions/leap
+  # A single action for all supported Leap releases.
+  # GET /distributions/leap/(:version)
```

Comment:
[nitpick] I find this message is not needed (self-explanatory in the routes) and that if it would be needed, it is should be in the routes file and not in the controller.